README.md

```diff
@@ -126,16 +138,17 @@ Feel free to check the [issues page](../../issues/).
 
 If you like this project just give it a star ⭐️.
 
-Everybody is welcome to suggest changes in the Bookstore project.
+Everybody is welcome to suggest changes to the Bookstore project.
 
 In order to do it, fork this repository, create a new branch and open a Pull Request from your branch
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-I would like to thank Microverse
+I would like to thank Microverse for and the designer at  <a href="https://www.behance.net/sakwadesignstudio">designer</a>
 
 ## 📝 License <a name="LICENSE"></a>
 
-This project is [MIT](./LICENSE) licensed.
+The content of this repository is licensed under the MIT License. Feel free to use, modify, and distribute the code following the terms of the license.
+Thank you for visiting GitHub repository.<a href="https://choosealicense.com/licenses/mit/">mit licence</a>
```

Comment:
- [ ] As previously requested, please ensure that the "mit license" link directs users to the "MIT.md" file within the repository, rather than an external MIT license website.